minecraft/networking/types.py

```diff
@@ -213,3 +223,28 @@ def read(file_object):
     @staticmethod
     def send(value, socket):
         socket.send(uuid.UUID(value).bytes)
+
+
+class Position(Type):
+    @staticmethod
+    def read(file_object):
+        location = UnsignedLong.read(file_object)
+        x = int(location >> 38)
+        y = int((location >> 26) & 0xFFF)
+        z = int(location & 0x3FFFFFF)
+
+        if x >= pow(2, 25):
+            x -= pow(2, 26)
+
+        if y >= pow(2, 11):
+            y -= pow(2, 12)
+
+        if z >= pow(2, 25):
+            z -= pow(2, 26)
+
+        return {'x': x, 'y': y, 'z': z}
```

Comment:
Creating a `Position` class would have the additional benefit that it contains all the fields required for `ClientExplosion.Record`, so that could simply subclass Position and pass.